ansible/roles/jupyterhub/files/jupyterhub_config_base.py

```diff
@@ -136,9 +136,13 @@
 mnt_root = os.environ.get('MNT_ROOT')
 # Mount volumes
 c.DockerSpawner.volumes = {
-    f'{mnt_root}/{org_name}' + '/home/{raw_username}': notebook_dir,
+    f'{mnt_root}/{org_name}' + '/home/{raw_username}': notebook_dir,    
```

Comment:
Did the linter pick up on this spacing at the EOL?

Reply:
In my case the post-commit action never works... i have to pay more attention with this..